cmd/srcd/cmd/gitbase_back_comp_test.go

```diff
@@ -0,0 +1,312 @@
+	require.NoError(err)
+
+	// [previous version] srcd sql "select * from repositories"
+	buf, err := s.runSQL(s.PrevCmd, "select * from repositories")
+	require.NoError(err)
+
+	expected := `+---------------+
+| REPOSITORY ID |
++---------------+
+| repo-0        |
+| repo-1        |
+| repo-2        |
++---------------+`
+	require.NoError(cmdtest.AreSQLOutputEqual(buf.String(), expected))
+
+	// [previous version] srcd sql "SHOW INDEXES"
+	buf, err = s.runSQL(s.PrevCmd, "SHOW INDEX FROM repositories")
+
+	require.NoError(err)
+	expected = `+-------+------------+----------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+---------+------------+
+| TABLE | NON UNIQUE | KEY NAME | SEQ IN INDEX | COLUMN NAME | COLLATION | CARDINALITY | SUB PART | PACKED | NULL | INDEX TYPE | COMMENT | INDEX COMMENT | VISIBLE | EXPRESSION |
++-------+------------+----------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+---------+------------+
++-------+------------+----------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+---------+------------+
+`
+	require.NoError(cmdtest.AreSQLOutputEqual(buf.String(), expected))
+
+	// [previous version] srcd sql "CREATE INDEXES"
+	_, err = s.runSQL(s.PrevCmd, "CREATE INDEX repo_idx ON repositories USING pilosa (repository_id)")
+	require.NoError(err)
+	// wait a bit for index to be ready
+	time.Sleep(1 * time.Second)
+
+	// [previous version] srcd sql "SHOW INDEXES"
+	buf, err = s.runSQL(s.PrevCmd, "SHOW INDEX FROM repositories")
+	require.NoError(err)
+	expected = `+--------------+------------+----------+--------------+----------------------------+-----------+-------------+----------+--------+------+------------+---------+---------------+---------+------------+
+|    TABLE     | NON UNIQUE | KEY NAME | SEQ IN INDEX |        COLUMN NAME         | COLLATION | CARDINALITY | SUB PART | PACKED | NULL | INDEX TYPE | COMMENT | INDEX COMMENT | VISIBLE | EXPRESSION |
++--------------+------------+----------+--------------+----------------------------+-----------+-------------+----------+--------+------+------------+---------+---------------+---------+------------+
+| repositories |          1 | repo_idx |            1 | repositories.repository_id | NULL      |           0 |        0 | NULL   |      | pilosa     |         |               | YES     | NULL       |
++--------------+------------+----------+--------------+----------------------------+-----------+-------------+----------+--------+------+------------+---------+---------------+---------+------------+
+`
+	require.NoError(cmdtest.AreSQLOutputEqual(buf.String(), expected))
+
+	// [previous version] srcd sql "select * from repositories"
+	buf, err = s.runSQL(s.PrevCmd, "select * from repositories")
+	require.NoError(err)
+
+	expected = `+---------------+
```

Comment:
Is this not the same than https://github.com/src-d/engine/pull/363/files#diff-0a916376db6c11d729358f6303375940R206 ?
Why is it needed? could it be different after the index creation?
Is it needed this test for the purpose of the backwardcompatibility test?

Reply:
mhmmm I think that at some point during the development I thought of checking the indexes explicitly. Given that also @smacker pointed out checking the index explicitly as you, I'm gonna address it 👍.